### DISCUSSION TITLE
May rowkey be duplicated in different db?

### DISCUSSION THREAD

**Discussion by zhuangqh** [apache/incubator-seata#4942] [Q&A] [answered]
As I know, in AT mode. The global lock key is `getRowKey(rowLock.getResourceId(), rowLock.getTableName(), rowLock.getPk()));`

There are two totally different db, may their lockkey be the same? Because the resourceId may be the same?

在AT模式中，全局锁的key是resourceId+tablename+pk。有没可能两个完全不相关的db，他们的锁key是一样的，因为resourceid可能是一样，比如jdbc里的地址是虚地址？ 这会导致两个无关的应用抢同一个锁。
请问seata是如何考虑这个问题的，或者我理解有误。谢谢

**Comment by zhuangqh** [comment]
@a364176773 @slievrly

**Comment by funky-eyes** [comment, marked as answer]
jdbc:mysql://127.0.0.1:3306/seata-order^^^order^^^2188
你觉得ip+dbname会出现相同mysqlip上有2个dbname一样几率大吗？
